js/gated-content/src/views/VideoPage.vue

```diff
@@ -7,32 +7,96 @@
     <div v-else-if="error">Error loading</div>
     <div v-else>
-      <h1>{{ video.attributes.title }}</h1>
-      <div v-html="video.attributes.field_gc_video_description.processed"></div>
-      <div>Category: {{ video.attributes.field_gc_video_category.name }}</div>
-      <div>Level: {{ video.attributes.field_gc_video_level.name }}</div>
-      <div>Equipment:
-        <ul>
-          <li v-for="equip in video.attributes.field_gc_video_equipment"
-              :key="equip.drupal_internal__tid">
-            {{ equip.name }}
-          </li>
-        </ul>
+      <div class="video">
+        <LazyYoutubeVideo
+          :src="'https://www.youtube.com/embed/' + video.attributes.field_gc_video_media.field_media_video_id"
+        />
       </div>
-      <div>Instructor: {{ video.attributes.field_gc_video_instructor }}</div>
-      <div>Duration: {{ video.attributes.field_gc_video_duration }}</div>
-      <hr />
-      <pre>Media: {{ video.attributes.field_gc_video_media }}</pre>
-      <pre>Video: {{ video.attributes }}</pre>
+      <div class="video-footer">
+        <div>
+          <div class="video-footer__title">{{ video.attributes.title }}</div>
+          <div class="video-footer__description"
+               v-html="video.attributes.field_gc_video_description.processed"
+          ></div>
+        </div>
+        <div>
+          <div class="video-footer__block">
+            <div class="video-level">{{ video.attributes.field_gc_video_level.name | first_letter }}</div>
+            {{ video.attributes.field_gc_video_level.name | capitalize }}
+          </div>
+          <div class="video-footer__block">
+            {{ video.attributes.field_gc_video_instructor }}
+          </div>
+          <div class="video-footer__block">
+            {{ video.attributes.field_gc_video_category.name }}
+          </div>
+          <div class="video-footer__equipment">Equipment:
+            <ul>
+              <li v-for="equip in video.attributes.field_gc_video_equipment"
+                  :key="equip.drupal_internal__tid">
+                {{ equip.name }}
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="video-category">
+        < {{ video.attributes.field_gc_video_category.name }}
+      </div>
+
+      <div class="video-up-next">
+        <div class="header">
+          UP NEXT
+        </div>
+
+        <div class="list">
+          <div>
+            <div class="preview">
+              <button type="button" aria-label="Play video" class="y-video__button"><svg viewBox="0 0 68 48" width="100%" height="100%"><path d="M66.5 7.7c-.8-2.9-2.5-5.4-5.4-6.2C55.8.1 34 0 34 0S12.2.1 6.9 1.6c-3 .7-4.6 3.2-5.4 6.1a89.6 89.6 0 0 0 0 32.5c.8 3 2.5 5.5 5.4 6.3C12.2 47.9 34 48 34 48s21.8-.1 27.1-1.6c3-.7 4.6-3.2 5.4-6.1C68 35 68 24 68 24s0-11-1.5-16.3z" class="y-video__button-shape"></path><path d="M45 24L27 14v20" class="y-video__button-icon"></path></svg></button>
```

Comment:
Looks like we use this SVG several times, let's move this to separated small component and reuse everywhere. (I think this will be in live stream too)